### PR TITLE
Add Haptic Feedback (vibration) for entering and exiting full screen mode

### PR DESCRIPTION
**Pull request by Natsu6767**

### CODE REVIEW

File: app/src/main/java/com/sdsmdg/harjot/MusicDNA/Activities/HomeActivity.java [hunk 1]
```diff
@@ -78,13 +79,18 @@
...
 import com.sdsmdg.harjot.MusicDNA.Fragments.AllPlaylistsFragment.AllPlaylistsFragment;
 import com.sdsmdg.harjot.MusicDNA.Fragments.EditSongFragment.EditLocalSongFragment;
 import com.sdsmdg.harjot.MusicDNA.Fragments.EqualizerFragment.EqualizerFragment;
 import com.sdsmdg.harjot.MusicDNA.Fragments.FavouritesFragment.FavouritesFragment;
 import com.sdsmdg.harjot.MusicDNA.Fragments.FolderContentFragment.FolderContentFragment;
-import com.sdsmdg.harjot.MusicDNA.Fragments.AllFoldersFragment.FolderFragment;
+import com.sdsmdg.harjot.MusicDNA.Fragments.LocalMusicFragments.AlbumFragment;
+import com.sdsmdg.harjot.MusicDNA.Fragments.LocalMusicFragments.ArtistFragment;
+import com.sdsmdg.harjot.MusicDNA.Fragments.LocalMusicFragments.LocalMusicFragment;
+import com.sdsmdg.harjot.MusicDNA.Fragments.LocalMusicFragments.LocalMusicViewPagerFragment;
```
Comment (kdexd) [inline app/src/main/java/com/sdsmdg/harjot/MusicDNA/Activities/HomeActivity.java]: Usually one should avoid unrelated changes in a PR, for a third person referring the purpose of this PR, such changes are noisy. Please undo these import reorders in this PR and include only the additions which are required for haptic feedback.

This shall also help you to see amending and force pushing commits if you haven't tried before.

Comment (Natsu6767) [reply app/src/main/java/com/sdsmdg/harjot/MusicDNA/Activities/HomeActivity.java]: Oh I didn't realize that. I only added the haptic feedback lines of code. It happened by itself, I think. I will fix it.

**Comment by Natsu6767** [comment]
I have fixed the order of the Import Statements that I had messed up initially.
